From de3e0a3b3a2fcdb1a062dd213a4f7933c8c5f7c9 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Thu, 2 Apr 2020 20:12:30 +0800
Subject: [PATCH] ARM: dts: rockchip: Enable rng for rv11xx-evb-v10

Before:
[  134.183519] random: crng init done
[  134.183851] random: 1 urandom warning(s) missed due to ratelimiting

After:
[    0.190031] random: crng init done

Change-Id: I80b876a3c227646bd4fcdd36d1a975cbc7e27d85
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm/boot/dts/rv11xx-evb-v10.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
index af813d1b3b2b..590ff116f349 100644
--- a/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
+++ b/arch/arm/boot/dts/rv11xx-evb-v10.dtsi
@@ -948,6 +948,10 @@
 	status = "okay";
 };
 
+&rng {
+	status = "okay";
+};
+
 &saradc {
 	status = "okay";
 	vref-supply = <&vcc_1v8>;
-- 
2.35.3

